File: Ejercicio/c sharp/MetodoConParametro/ParametroDiferentes.cs
using System;

class Ejemplo4 {
    static void MostrarInfo(string nombre, int edad, double altura) {
        Console.WriteLine($"{nombre} tiene {edad} años y mide {altura} metros.");
    }

    static void Main() {
        MostrarInfo("Valery", 22, 1.68);
    }
}

File: Ejercicio/GO/MetodoConParametro/ParametroDiferentes.go
package main
import "fmt"

func mostrarInfo(nombre string, edad int, altura float64) {
    fmt.Printf("%s tiene %d años y mide %.2f metros.\n", nombre, edad, altura)
}

func main() {
    mostrarInfo("Valery", 22, 1.68)
}
